ts/state/ducks/search.ts

```diff
@@ -215,15 +215,6 @@ async function queryConversationsAndContacts(
     }
   }
 
-  // Inject synthetic Note to Self entry if query matches localized 'Note to Self'
```

Comment:
why was this removed?

Reply:
because I thought that was not something we keep. I will correct it